### PR TITLE
fix GRASS v_net_visibility module

### LINKED ISSUE #41030: Duplicate parameter coordinates registered for alg v.net.visibility
**Describe the bug**

The Log Messages panel shows the following warning:

`WARNING    Duplicate parameter coordinates registered for alg v.net.visibility`

**QGIS and OS versions**

QGIS 3.10.13 - QGIS 3.16.2

**Additional context**

This issue was fixed in 3.2 with 5436ce9a3583fa77e335931dd69828ca950c0894 (https://github.com/qgis/QGIS/pull/6649) and then reintroduced since 3.10.11 and 3.16.0 with 3119eb7de2f2d1bf27001d6b5160af9295b3b355 (https://github.com/qgis/QGIS/pull/38966)

### PR DESCRIPTION
**Pull request by gioman**
Fixes https://github.com/qgis/QGIS/issues/41030

### CODE REVIEW

**Comment by nyalldawson** [comment]
@agiudiceandrea are you happy with this fix?

**Comment by agiudiceandrea** [comment]
I think it could be better to use QgsProcessingParameterString for the "coordinates" parameter, otherwise it wouldn't be possible to insert the coordinates of more then one point.

File: python/plugins/processing/algs/grass7/description/v.net.visibility.txt [hunk 1]
```diff
@@ -3,6 +3,5 @@ Performs visibility graph construction.
 Vector (v.*)
 QgsProcessingParameterFeatureSource|input|Input vector line layer (arcs)|-1|None|False
 QgsProcessingParameterPoint|coordinates|Coordinates|None|True
```
Comment (agiudiceandrea) [inline python/plugins/processing/algs/grass7/description/v.net.visibility.txt:5]:
```suggestion
QgsProcessingParameterString|coordinates|Coordinates|None|False|True
```

**Review by agiudiceandrea — CHANGES_REQUESTED** [review]

**Comment by gioman** [comment]
> I think it could be better to use QgsProcessingParameterString for the "coordinates" parameter, otherwise it wouldn't be possible to insert the coordinates of more then one point.

@agiudiceandrea done

**Pull request merged**